Develop

trekab opened the pull request:
I have cloned the newsweek.com home page in this branch. I used bootstrap where possible and some custom styles.

trekab committed:
Add Stickler's config files for CSS

trekab committed:
Add more page elements and position them with bootstrap

trekab committed:
Finish page structure positioned with bootstrap

trekab committed:
Add typography styles for most headings

trekab committed:
Style main section elements

trekab committed:
Style the footer

trekab committed:
Fix search input in top nav

trekab committed:
Update REAME file

trekab committed:
Update stylelint config file

trekab committed:
Fix linter errors

trekab committed:
Work on linter errors in genericon.css file

trekab committed:
Fix space errors flaged by stickler

trekab committed:
Fix more stickler errors

trekab committed:
Fix space errors in css

trekab committed:
Use 2 spaces for indentation

trekab committed:
Fix pseudo selectors in genericon.css file

trekab committed:
Fix main styles linter errors

trekab committed:
Fix more linter errors

trekab committed:
Fix css linter errors

ebukaume commented:
Hi @trekab :wave: 

You have really done a nice job so far. However, there are a few places we need to straighten things out.

- [ ]  The project has a descriptive README.md (You can refer to [this repo](https://github.com/othneildrew/Best-README-Template) as a guide.)

**Navigation Bar**
Something weird is happening with your breakpoints. Please fix them.

![review-trekab](https://user-images.githubusercontent.com/22497270/66779678-b63e9200-eec6-11e9-953b-c69e27fb94ff.png)


**Main content**
Your breakpoints are quite off from what is obtainable on [News Week's website](https://www.newsweek.com/). Observe closely how the are elements change and reorder at (_see how the `Top Story` always stays relevantly on top_):

_**Large**_
![newsweek1](https://user-images.githubusercontent.com/22497270/66780069-ce62e100-eec7-11e9-9bb2-e20ff0786149.png)

_**Medium**_
![newsweek2](https://user-images.githubusercontent.com/22497270/66780084-d91d7600-eec7-11e9-8662-b6b303056d18.png)

**_Small_**
![newsweek3](https://user-images.githubusercontent.com/22497270/66780101-e470a180-eec7-11e9-8498-62178aebdf21.png)

- [ ] Add [vieport metatag](https://getbootstrap.com/docs/4.3/getting-started/introduction/#responsive-meta-tag)

**Kindly submit another code review request once you have fixed the issues mentioned above**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

trekab committed:
Redesign page and make it responsive

Redesign page structure and make more use of bootstrap classes. Use font awesome for icons and refactor custom css file. Make use of
web hosted images for most images.

trekab committed:
Refine custom css

trekab committed:
Update readme file

trekab committed:
Fix linter errors

trekab commented:
I have managed to work on the page responsiveness and add more bootstrap classes to the project.

suretrust commented:
**Hi @trekab :raised_hands:** 

Great job! **Let's get this merged! :tada:** 

_Good luck with other projects and happy coding!_

P.S: Don't forget to close this [issue](https://github.com/trekab/using-bootstrap/issues/2).
